app/controllers/idv/review_controller.rb

```diff
@@ -30,7 +32,6 @@ def confirm_current_password
     end
 
     def new
-      @applicant = idv_session.applicant
```

Comment:
I think we can also remove the assign in the spec:

https://github.com/18F/identity-idp/blob/e7e88a29ed4fc3487e2495e7640714100a0d60c8/spec/views/idv/review/new.html.erb_spec.rb#L14-L24